source/utilities.tex

```diff
@@ -14759,6 +14826,150 @@
+as described by table 12 of UAX \#44, or
+
+\item
+\placeholder{CE} is not a Unicode encoding and
+\placeholder{C} is one of an implementation-defined set
+of separator or non-printable characters
+\end{itemize}
+
+then the sequence \tcode{$\backslash$u\{\placeholder{hex-digit-sequence}\}}
+is appended to \placeholder{E},
+where \placeholder{hex-digit-sequence}
+is the shortest hexadecimal representation
+of \placeholder{C} using lower-case hexadecimal digits.
+
+\item
+Otherwise, \placeholder{C} is appended to \placeholder{E}.
+\end{itemize}
+
+\item
+Otherwise, if \placeholder{X} is a shift sequence,
+the effect on \placeholder{E} and further decoding of \placeholder{S}
+is unspecified.
+It is recommended that
+a shift sequence be represented in \placeholder{E}
+such that the original code unit sequence of \placeholder{S}
+can be reconstructed.
+
+\item
+Otherwise (\placeholder{X} is a sequence of ill-formed code units),
+each code unit \placeholder{U} is appended to \placeholder{E} in order
+as the sequence \tcode{$\backslash$x\{\placeholder{hex-digit-sequence}\}},
+where \placeholder{hex-digit-sequence}
+is the shortest hexadecimal representation of \placeholder{U}
+using lower-case hexadecimal digits.
+\end{itemize}
+
+\item
+Finally, \unicode{0022}{quotation mark} (\tcode{"})
+is appended to \placeholder{E}.
+\end{itemize}
+%
+\begin{floattable}{Mapping of characters to escape sequences}{format.escape.sequences}{ll}
+\topline
+\lhdr{Character} & \rhdr{Escape sequence} \\ \rowsep
+\unicode{0009}{character tabulation} &
+\tcode{$\backslash$t}
+\\ \rowsep
+%
+\unicode{000a}{line feed} &
+\tcode{$\backslash$n}
+\\ \rowsep
+%
+\unicode{000d}{carriage return} &
+\tcode{$\backslash$r}
+\\ \rowsep
+%
+\unicode{0022}{quotation mark} &
+\tcode{$\backslash$"}
+\\ \rowsep
+%
+\unicode{005c}{reverse solidus} &
+\tcode{$\backslash\backslash$}
+\\
+\end{floattable}
+
+\pnum
+The escaped string representation of a character \placeholder{C}
+is equivalent to the escaped string representation
+of a string of \placeholder{C}, except that:
+
+\begin{itemize}
+\item
+the result starts and ends with \unicode{0027}{apostrope} (\tcode{'})
+instead of \unicode{0022}{quotation mark} (\tcode{"}), and
+\item
+if \placeholder{C} is \unicode{0027}{apostrope},
+the two characters \tcode{$\backslash$'} are appended to \placeholder{E}, and
+\item
+if \placeholder{C} is \unicode{0022}{quotation mark},
+then \placeholder{C} is appended unchanged.
+\end{itemize}
+
+%% FIXME: Example is incomplete; s2 and s6 are missing below;
+%% FIXME: their Unicode characters break the build.
+\begin{example}
+\begin{codeblock}
+string s0 = format("[{}]", "h\tllo");                   // \tcode{s0} has value: \tcode{[h    llo]}
+string s1 = format("[{:?}]", "h\tllo");                 // \tcode{s1} has value: \tcode{["h$\backslash$tllo"]}
+string s3 = format("[{:?}] [{:?}]", '\'', '"');         // \tcode{s3} has value: \tcode{['$\backslash$'', '"']}
```

Comment:
All the backslashes in this example should be textbackslash in code font.

Reply:
fixed